catalog/video/video-mpegts-stream/src/main/java/org/codice/alliance/video/stream/mpegts/UdpStreamMonitor.java

```diff
@@ -546,6 +546,10 @@ public void setElapsedTimeRolloverCondition(Long milliseconds) {
         @Override
         public void run() {
 
+            LOGGER.debug("starting udp listening thread: address={} port={}",
+                    monitoredAddress,
+                    monitoredPort);
```

Comment:
Perhaps include the other conditions - rolling over by time or size and what those limits are?